carapace-server/src/main/java/org/carapaceproxy/server/mapper/StandardEndpointMapper.java

```diff
@@ -456,13 +454,14 @@ public MapResult map(HttpRequest request, String userId, String sessionId, Backe
                                 .setCustomHeaders(customHeaders);
                     }
                 }
+                // none of selected backends available
+                if (!selectedBackends.isEmpty()) {
+                    return MapResult.INTERNAL_ERROR(route.getId());
```

Comment:
Not sure which is better: NOT_FOUND or INTERNAL_ERROR
@eolivelli

Reply:
@pv3nturi it should return "http 503 (service not available)". If that't not possible, lets return http 500 (INTERNAL_ERROR, right?) and fix that in a future issue

Reply:
"http 503 (service not available)" is currently unavailable... let's fix it in the future